package.json

```diff
@@ -3,22 +3,22 @@
   "version": "0.2.1",
   "license": "MIT",
   "private": false,
-  "main": "src/references.js",
+  "main": "dist/src/index",
```

Comment:
You are thinking it should be `"dist/index"` instead? 🤔

Reply:
Yeah, `src/` is typically built into `dist/` is what I've seen

Reply:
I'm pretty sure that `tsc` is keeping the folder structure inside `/dist` because we are referencing `references.json` from ../deployments on some of the ts files under `/src`. There might be ways to avoid this (path mappings?) but it might be worth doing this later.

Reply:
Right, that sounds likely